MVCDemos/Controllers/ContextMenuController.cs

```diff
@@ -1,6 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
+﻿using DevExtreme.AspNet.Data;
+using DevExtreme.AspNet.Mvc;
+using DevExtreme.MVC.Demos.Models.SampleData;
+using Newtonsoft.Json;
```

Comment:
Is it really necessary here? Does not the demo work without it?

Reply:
[3e130c0](https://github.com/DevExpress/devextreme-demos/pull/3203/commits/3e130c098fbe91e5583e340f3b1092a8140650ae)